pkg/cmd/version.go

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright The Dragonfly Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/dragonflyoss/Dragonfly/pkg/printer"
+	"github.com/dragonflyoss/Dragonfly/version"
+
+	"github.com/spf13/cobra"
+)
+
+// versionLongTempl is used to describe version command in detail and auto generate command doc.
+var versionLongTempl = "Display the version and build information of Dragonfly %s, " +
+	"including GoVersion, OS, Arch, Version, BuildDate and GitCommit."
+
+func NewVersionCommand(name string) *cobra.Command {
```

Comment:
Please do not forget to add the `Example` field. And it is be used to generate docs.